src/env.js

```diff
@@ -30,6 +30,14 @@ const nonemptystr = makeValidator((v) => {
   return trimmed;
 });
 
+const jsonFile = makeValidator((v) => {
+  if (v === undefined || v === null || v === '') {
+    throw new Error('Expected a non-empty string');
+  }
+  // eslint-disable-next-line import/no-dynamic-require
+  return require(v); // eslint-disable-line global-require
```

Comment:
I don't think the validator should do the require, should it?
It should only be concerned with validating that it is a file and it exists

Reply:
I did this intentionally, as requiring will also parse the file as js/json. Either having a missing file or a file with invalid json will cause an unhandled rejection from the google api client, so I thought it prudent to catch that up front.